Automattic/wp-calypso
# Generated design picker: Fetch recipes for generated designs via API

#### Changes proposed in this Pull Request

This PR retrieves recipes for generated designs from the backend instead of a local file. This allows us to manage the list of recipes without needing to update the front-end.  

#### Testing instructions



* Make sure you can access the generated design picker screen by:
  * Having the necessary flags enabled.
  * Having a vertical selected from the vertical question screen.
* Head to `/setup/intent?siteSlug=${site_slug}`, and click on the "Start building" button.
* Expect the list of generated designs to match the list from the `block-recipes` [API endpoint](https://public-api.wordpress.com/wpcom/v2/block-recipes).



Related to #62237
